src/intro/tour.rst

```diff
@@ -58,7 +63,7 @@ All we added to the previous request is the _all_dbs string.
 
 The response should look like::
 
-    ["_replicator","_users"]
+    []
```

Comment:
This means you didn't finish installation correctly. See http://docs.couchdb.org/en/stable/install/setup.html for the steps you missed.

Absolutely the response should be `["_replicator","_users"]`. If you want to put an admonishment in here with a link to the doc page I mentioned above, go for it. ([Use the RST syntax for linking](http://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#ref-role), not a link to the URL, please.)

Reply:
Ok, I see. What I did is just pull `couchdb:2.2.0` from docker hub, `run` it, and since I was able to read and write data, I assumed things were setup correctly, so adding a link to the setup page would definitely be a good idea I think.

That being said, after doing what I have been advised to in the setup doucment I will see the following:

```
➜  ~ curl http://127.0.0.1:5984/_all_dbs
["_global_changes","_replicator","_users"]
```

which still does not match the documented response. Should this be edited in order to contain the `_global_changes`?

Reply:
Yes, sorry, `_global_changes` can be added to the response. Its creation is optional, but useful to highlight.

The Docker container doesn't do the initial creation of databases because that cannot be done until all nodes are joined to the cluster. If you're using Docker to create a 3-node cluster, you wouldn't want it to incorrectly create those databases locally on each node before you have a chance to merge the nodes together into a single cluster. It's a chicken-and-egg problem ;)

Reply:
@m90 Would you have time to get this cleaned up soon? We are hoping to release 2.3.0 as soon as next week.

Reply:
@wohali Done so.